packages/@aws-cdk/aws-s3-deployment/test/lambda/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM public.ecr.aws/lambda/python:latest
+
+# add everything to /opt/awscli (this is where `aws` is executed from)
+ADD . /opt/awscli
+
+# install boto3, which is available on Lambda
+RUN pip3 install boto3
+
+# run tests
+WORKDIR /opt/awscli
+RUN ["python3", "./test.py"]
```

Comment:
Any specific reason you run this on build? Seems more appropriate as the container command. This also makes it impossible to debug failing tests inside the container.

Reply:
It's a common practice to use `docker build .` as a way to simply execute a bunch of commands inside a container. Otherwise, we'll have to `docker build .` and then `docker run`, which is not needed.